API-strategie-modules/transport-security/transport-security.md

```diff
@@ -67,17 +81,24 @@ The headers below are only intended to provide additional security when response
 
 In addition to the above listed HTTP security headers, web- and browser-based applications SHOULD apply Subresource Integrity [SRI](https://www.w3.org/TR/SRI/). When using third-party hosted contents, e.g. using a Content Delivery Network, this is even more relevant. While this is primarily a client implementation concern, it may affect the API when it is not strictly segregated or for example when shared supporting libraries are offered.
 
+**How to test**
+The precense of the mandatory security headers can be tested in an automated way. A test client makes a call to the API root. The response is tested for the precense of mandatory headers.
```

Comment:
Boven de table met headers wordt `SHOULD` gebruikt en geen `MUST`. Op zich logisch want `Cache-Control: no-store` is niet wenselijke voor niet gevoelige informatie.

Denk dat het goed om nog eens naar de lijst met headers en de waarde te kijken. Nu is het not te 'vrij' welke headers echt nodig zijn en welke waarde de moeten hebben of bevatten. 

`Strict-Transport-Security` zou een losse regel kunnen worden. Je kan testen op `max-age` voldoende is. Als het domein preloaded is, dan is deze header niet nodig. Zoals bij `.dev` het geval is.